src/fireBaseAuth.js

```diff
@@ -0,0 +1,37 @@
+import { auth } from './firebase';
+import {
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+  signOut
+} from 'firebase/auth';
+
+async function signUp(email, password) {
+  try {
+    const user = await createUserWithEmailAndPassword(auth, email, password);
+    console.log(user)
+    const userName = user.user.uid;
+    return userName;
+  } catch (error) {
+    throw error;
+  }
+}
+async function signIn(email, password) {
+  try {
+    const user = await signInWithEmailAndPassword(auth, email, password);
+    console.log(user.user.uid)
+    const userName = user.user.uid;
+    return userName;
+  } catch (error) {
+    console.log(error)
```

Comment:
You can use `console.error` for handling errors.

Reply:
Yes thank you